src/pages/IstioConfigDetails/IstioObjectDetails/VirtualServiceOverview.tsx

```diff
@@ -115,6 +115,32 @@ class VirtualServiceOverview extends React.Component<VirtualServiceProps> {
     );
   }
 
+  generateEmptyGateways() {
+    return (
+      <>
+        <Text component={TextVariants.h3}>Gateways</Text>
+        <ul className={'details'}>
+          <li key={'gateway_mesh'}>
+            <div>
+              mesh
+              <Tooltip
+                position={TooltipPosition.right}
+                content={
+                  <div style={{ textAlign: 'left' }}>
+                    When this field is omitted, the default gateway (mesh) will be used, which would apply the rule to
```

Comment:
Just a minor suggestion, instead of saying "this", what about saying "When the gateways field is ommited", because I was testing this and it was a little confusing, about what field, because the tooltip is on the "mesh" gateway.

Reply:
+1, fixed